app/uk/gov/hmrc/cdsreimbursementclaim/models/claim/CompleteClaim.scala

```diff
@@ -62,7 +63,7 @@ object CompleteClaim {
     maybeDuplicateDisplayDeclaration: Option[DisplayDeclaration],
     importerEoriNumber: Option[CompleteImporterEoriNumberAnswer],
     declarantEoriNumber: Option[CompleteDeclarantEoriNumberAnswer],
-    completeClaimsAnswer: CompleteClaimsAnswer
+    completeClaimsAnswer: ClaimsAnswer
```

Comment:
rename completeClaimsAnswer to claimsAnswer - should it be claimAnswers or claimsAnswer?

Reply:
done